cpu/cc2538/clock.c

```diff
@@ -61,8 +61,20 @@
 
 #include <stdint.h>
 /*---------------------------------------------------------------------------*/
-#define RTIMER_CLOCK_TICK_RATIO (RTIMER_SECOND / CLOCK_SECOND)
-#define RELOAD_VALUE (125000 - 1)  /** Fire 128 times / sec */
+#define RTIMER_CLOCK_TICK_RATIO			(RTIMER_SECOND / CLOCK_SECOND)
```

Comment:
All these indentation problems come either from my emacs configuration or from the uncrustify script... May i assume that blanks are used for indentation (not tabs?)